app/javascript/crayons/formElements/RadioButton/RadioButton.jsx

```diff
@@ -0,0 +1,31 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+
+export const RadioButton = ({ id, value, className, checked }) => {
```

Comment:
@ludwiczakpawel, in regards to the `<RadioButton />` component, would it always be wrapped in a <FormField /> component? Just asking, because if that is the case, I will update the `<RadioButton />`.

Reply:
We discussed this on Zoom, so `<RadioButton />` should remain separate.